src/counters.c

```diff
@@ -511,6 +511,7 @@ static void *StatsWakeupThread(void *arg)
             /* assuming the assignment of an int to be atomic, and even if it's
              * not, it should be okay */
             tv->perf_public_ctx.perf_flag = 1;
+            tv->profile_flag = 1;
```

Comment:
I don't like how the detect specific logic flows into the core threading code here

Reply:
@regit what is the goal here? To sync threads on rule profile data at the same interval as normal stats? Or is the frequency not very important?